.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -22,15 +22,9 @@ body:
     id: solution
     attributes:
       label: Proposed solution
-      description: Please provide a clear and concise description of a way to accomplish what you want.
-      placeholder: Add an option so that when [...]  [...] will happen
+      description: A concise description of a way to accomplish what you want.
```

```suggestion
      description: Briefly describe a way to accomplish the goals of this feature request. 
```
don't like my wording either, but don't think we're being clear here on what we actually want.

---

I wonder if "proposed solution" and "prior art" could just be one.  i.e. 
"Suggested method to do what you are proposing, and examples of the feature in other software or publications"